<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SE_GeoCoordSysTransDialog</class>
 <widget class="QDialog" name="SE_GeoCoordSysTransDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地理坐标系转换工具</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>543</width>
       <height>185</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_InputData">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>输入数据目录</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="2">
          <widget class="QPushButton" name="Button_Open">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_OdataPath">
           <property name="text">
            <string>数据路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_InputDataPath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>输入shp数据所在目录</string>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_OutputData">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>输出数据目录</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_OutputDataPath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>地理坐标系转换结果保存路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_OutputPath">
           <property name="text">
            <string>保存路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="Button_Save">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_OutputSRS_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>源地理坐标系</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_CGCS2000_From">
           <property name="text">
            <string>CGCS2000坐标系</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_BJS54_From">
           <property name="text">
            <string>北京54坐标系</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_XAS80_From">
           <property name="text">
            <string>西安80坐标系</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton_WGS84_From">
           <property name="text">
            <string>WGS84坐标系</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButton_Foreign_From">
           <property name="text">
            <string>境外坐标系</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>参考椭球参数</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_OffsetX_8">
              <property name="text">
               <string>长半轴</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_SemiMajorAxis">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_OffsetXUnit_3">
              <property name="text">
               <string>米</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_OffsetX_7">
              <property name="text">
               <string>短半轴</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLineEdit" name="lineEdit_SemiMinorAxis">
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_OffsetXUnit_4">
              <property name="text">
               <string>米</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_OutputSRS">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>目标地理坐标系</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_CGCS2000_To">
           <property name="text">
            <string>CGCS2000坐标系</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_BJS54_To">
           <property name="text">
            <string>北京54坐标系</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_XAS80_To">
           <property name="text">
            <string>西安80坐标系</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="radioButton_WGS84_To">
           <property name="text">
            <string>WGS84坐标系</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox_OffsetSetting">
        <property name="title">
         <string>转换参数设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>平移</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_dX">
              <property name="text">
               <string>△X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_dX">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_OffsetXUnit">
              <property name="text">
               <string>米</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_dY">
              <property name="text">
               <string>△Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_dY">
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_OffsetYUnit">
              <property name="text">
               <string>米</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_dZ">
              <property name="text">
               <string>△Z</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_dZ">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_OffsetXUnit_2">
              <property name="text">
               <string>米</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>旋转</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_rY">
              <property name="text">
               <string>rY</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_rZ">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_rX">
              <property name="text">
               <string>rX</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_OffsetXUnit_6">
              <property name="text">
               <string>秒</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_rY">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_rZ">
              <property name="text">
               <string>rZ</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_OffsetXUnit_7">
              <property name="text">
               <string>秒</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_rX">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_OffsetXUnit_8">
              <property name="text">
               <string>秒</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>缩放</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_K">
              <property name="text">
               <string>K</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_K">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_KUnit">
              <property name="text">
               <string>PPM（百万分之一）</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QGroupBox" name="groupBox_log_level">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>日志设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_log_level_name">
           <property name="text">
            <string>日志等级</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox_level">
           <item>
            <property name="text">
             <string>消息</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>警告</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>关闭</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>调试</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>详细</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>一般错误</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>致命错误</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>688</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_log_level_name_2">
           <property name="text">
            <string>汇总日志保存路径</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLineEdit" name="lineEdit_summary_log_saving_path">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="Button_summary_log_saving_path">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="Button_OK">
     <property name="text">
      <string>开始转换</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Button_Cancel">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
